Packs/Orca/Integrations/Orca/Orca.py

```diff
@@ -169,15 +247,15 @@ def get_incident_from_alert(alert: Dict[str, Any]) -> Dict[str, Any]:
         'name': alert.get('state', {}).get('alert_id'),
         'occurred': last_seen_time,
         'rawJSON': json.dumps(alert),
-        'severity': map_orca_score_to_demisto_score(orca_score=alert.get('state', {}).get('score'))
```

Comment:
why did you change it from score to risk_level?

Reply:
`score` will deprecated soon.